docker/Dockerfile.op-move

```diff
@@ -28,7 +28,7 @@ RUN --mount=type=cache,target=/root/.cargo/registry \
 # RocksDB breaks the build with libclang.so not found https://github.com/apache/skywalking/issues/10439
 # Aptos-core breaks the build with `disable_lifo_slot` not found https://github.com/aptos-labs/aptos-core/issues/5655 \
     RUSTFLAGS="-C target-feature=-crt-static --cfg tokio_unstable" \
-    cargo build --bin op-move --release --features storage \
+    cargo build --bin op-move --release --features storage-lmdb \
```

Comment:
We should organize these feature names and package names later. Rocks DB uses just `storage` as the feature name, but LMDB has the full name. Also the packages are called with `heed` not LMDB.

Reply:
It's true that Rocks DB should also have the feature flag name extended to be consistent and transparent on the meaning of these flags.

My thoughts on the naming were: Looking at it from the user's perspective, I care about using `lmdb`, rather than the fact that it is ported through `heed`.